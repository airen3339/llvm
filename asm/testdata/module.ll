source_filename = "foo.c"
target datalayout = "e"
target triple = "x86_64-unknown-linux"

module asm "foo"

%t1 = type i32

%t2 = type opaque

$com1 = comdat any
$com2 = comdat exactmatch
$com3 = comdat largest
$com4 = comdat noduplicates
$com5 = comdat samesize

@g1 = external global i32

@g2 = global i32 0

declare void @exit(i32 %staus) #0

declare i32 @printf(i8*, ...)

define void @f1() {
	ret void
}

define i32 @f2(i32 %x, i32 %y) {
	ret i32 42
}

attributes #0 = { noreturn }

!foo = !{!0}

!0 = !{!"foo"}
