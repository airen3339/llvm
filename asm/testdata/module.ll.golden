target datalayout = "e"
target triple = "x86_64-unknown-linux"

%t1 = type i32

%t2 = type opaque

@g1 = external global i32

@g2 = global i32 0

declare void @exit(i32 %staus)

declare i32 @printf(i8*, ...)

define void @f1() {
; <label>:0
	ret void
}

define i32 @f2(i32 %x, i32 %y) {
; <label>:0
	ret i32 42
}

!foo = !{!0}

!0 = !{!"foo"}
